cni/pkg/mount/mount.go

```diff
@@ -0,0 +1,52 @@
+package mount
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+)
+
+// RunSandboxed is an opinionated function to run a command in a lightweight sandbox.
+// This overrides files that are not desired in iptables execution (which runs on the node) by bind mounting them
+func RunSandboxed(lockFile string, args []string) error {
```

Comment:
```suggestion
func RunIptablesSandboxed(lockFile string, args []string) error {
```

To make it very explicit that this is only sandboxing iptables quirks?